src/parser/UpdateClause.h

```diff
@@ -4,18 +4,73 @@
 
 #pragma once
 
+#include "parser/Iri.h"
 #include "parser/SelectClause.h"
 #include "parser/SparqlTriple.h"
+#include "parser/data/GraphRef.h"
 #include "parser/data/Types.h"
 
+struct Load {
```

Comment:
These have very common names and should be in a namespace, e.g. `updateClause::`